packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -74,7 +74,9 @@ export class DocumentsService {
       leafClusterId: params.leafClusterId,
       kubeId: params.kubeId,
       kubeUri: options.kubeUri,
-      kubeConfigName: options.kubeConfigName || `${params.kubeId}-${unique(5)}`,
+      kubeConfigName:
+        options.kubeConfigName ||
+        `${params.rootClusterId}/${params.kubeId}-${unique(5)}`,
```

Comment:
I think it'd be a good idea to note here why we're prepending this with `rootClusterId`.